test/integration/bigint/test/index.test.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-env jest */
+/* global jasmine */
+import fs from 'fs-extra'
+import { join } from 'path'
+import {
+  fetchViaHTTP,
+  launchApp,
+  nextBuild,
+  nextStart,
+  killApp,
+  findPort,
+} from 'next-test-utils'
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000 * 60 * 2
+
+const appDir = join(__dirname, '..')
+const nextConfig = join(appDir, 'next.config.js')
+let appPort
+let app
+
+const runTests = () => {
+  it('should return 200', async () => {
+    const res = await fetchViaHTTP(appPort, '/api/bigint', null, {
+      method: 'GET',
+    })
+    expect(res.status).toEqual(200)
+  })
+
+  it('should return the BigInt result text', async () => {
+    const resText = await fetchViaHTTP(appPort, '/api/bigint', null, {
+      method: 'GET',
+    }).then(res => res.ok && res.text())
+    expect(resText).toEqual('3')
```

Comment:
BigInt is not JSON serializable from what i've found (https://github.com/tc39/proposal-bigint/issues/162, https://github.com/tc39/proposal-bigint/issues/24), so I'm just going to test that `(1n + 2n).toString() = 3`